### PR TITLE
CWS faqs removal and new CWS TOC

### PR DESCRIPTION
**Pull request by AmySteam**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

- Applies new TOC
- Removes final articles agreed upon
- Removes CWS FAQs
- Removes deprecated payment and licensing API pages

### CODE REVIEW

File: site/en/docs/webstore/account-deletion/index.md [hunk 1]
```diff
@@ -15,13 +15,22 @@ things you need to understand before you delete an account:
...
 - Account deletion is permanent. There is no "undo" for this action.
 - You can't delete an account that still has published items. If you have published items, make sure
-  to [unpublish][unpublish] them before you try and delete the account.
+  to [unpublish](#remove-listing) them before you try and delete the account.
 - Group publisher accounts can only be deleted by the owner or manager of the associated Google
   Group.
 
+### How to remove my listing from the store? {: #remove-listing }
+
+Visit the [developer dashboard][dev-dashboard]. Click the item you want to
```
Comment (jpmedley) [inline site/en/docs/webstore/account-deletion/index.md:24]: This should probably be steps, but I don't know if we have time.

**Review by jpmedley — APPROVED** [review]
Looks good. I left one comment you may want to follow up on.

File: site/en/docs/webstore/account-deletion/index.md [hunk 2]
```diff
@@ -15,13 +15,22 @@ things you need to understand before you delete an account:
 
 - Account deletion is permanent. There is no "undo" for this action.
 - You can't delete an account that still has published items. If you have published items, make sure
-  to [unpublish][unpublish] them before you try and delete the account.
+  to [unpublish](#remove-listing) them before you try and delete the account.
 - Group publisher accounts can only be deleted by the owner or manager of the associated Google
   Group.
 
+### How to remove my listing from the store? {: #remove-listing }
```
Comment (AmySteam) [inline site/en/docs/webstore/account-deletion/index.md:22]:
```suggestion
### How to remove my listing from the store {: #remove-listing }
```

File: site/en/docs/webstore/account-deletion/index.md (hunk 1 above)
Comment (AmySteam) [reply site/en/docs/webstore/account-deletion/index.md:24]: I'll do it on a separate PR on Monday

File: site/en/docs/webstore/account-deletion/index.md [hunk 3]
```diff
@@ -15,13 +15,22 @@ things you need to understand before you delete an account:
 
 - Account deletion is permanent. There is no "undo" for this action.
 - You can't delete an account that still has published items. If you have published items, make sure
-  to [unpublish][unpublish] them before you try and delete the account.
+  to [unpublish](#remove-listing) them before you try and delete the account.
 - Group publisher accounts can only be deleted by the owner or manager of the associated Google
   Group.
 
+### How to remove my listing from the store {: #remove-listing }
```
Comment (AmySteam) [inline site/en/docs/webstore/account-deletion/index.md:22]:
```suggestion
## How to remove a store listing item {: #remove-listing }
```

**Pull request merged**